specificationRepositoryConfiguration.json

```diff
@@ -13,8 +13,9 @@
       "configFilePath": "eng/automation/swagger_to_sdk_config.json"
     },
     "azure-sdk-for-js": {
-      "integrationRepository": "azure-sdk/azure-sdk-for-js",
-      "mainRepository": "Azure/azure-sdk-for-js"
+       "integrationRepository": "test-repo-billy/azure-sdk-for-js",
```

Comment:
please revert the change of this file. only change this file for test-repo-billy